src/pages/dashboard.tsx

```diff
@@ -1,26 +1,14 @@
 import { AppHeader, Loading } from "../components";
-import { signIn, signOut, useSession } from "next-auth/client";
-import { useEffect } from "react";
-import { useRouter } from "next/router";
 
 export default function Dashboard() {
-  const router = useRouter();
-  const [session, loading] = useSession();
-
-  useEffect(() => {
-    if (!session && !loading) router.push("/");
-  }, [session, loading]);
-
-  return session ? (
+  return (
     <main>
       <AppHeader
-        title={`Hello there, ${session.user.name}`}
+        title='Hello there'
         description="You can see your portfolios estimated value & progress below:"
         currentPage={{ label: "Home", path: "/dashboard" }}
         otherPages={[{ label: "Portfolio", path: "/dashboard/portfolio" }]}
```

Comment:
is the home and portfolio links intended to be this way ?
Home -> doesn't go back to root, but remain in dashboard

Reply:
Yeah, maybe it's the wrong term. But I thought `Home` would be the page where they can see all their growth charts, while `Portfolio` will be where they can find and add new stocks. Not sure if it's correct though. Feel free to change.

Reply:
it looks good for now. maybe after discussing the milestones and requirements we can think about this.
go ahead and click `squash and merge` make sure you write a good title and descriptive commit body